packages/typeless-sample-bot/src/bin/typeless-sample-bot.ts

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
```

Comment:
Adding a shebang and `bin` the main differences?

Reply:
Yeah, and there are some weirdnesses around module naming/paths and ESM, so I went ahead and separated it out with the right name.